### PR TITLE
MODINV-501 - Create Handler to generate Authorities from MARC Authority records

### PR DESCRIPTION
**Pull request by natali-zaitseva**
## Purpose
In the scope of handling MARC Authority creation via data-import we need to add a handler to map and create a corresponding Authority record and then send it to the mod-inventory-storage module.

## Approach

* added handler for MARC Authority records
* added external collection classes to work with inventory-storage
* added test
* updated raml-util to latest master version
* updated NEWS.md file

### CODE REVIEW

File: src/main/java/org/folio/inventory/dataimport/consumers/DataImportKafkaHandler.java [hunk 1]
```diff
@@ -18,6 +18,7 @@
 import org.folio.inventory.dataimport.handlers.actions.CreateHoldingEventHandler;
 import org.folio.inventory.dataimport.handlers.actions.CreateInstanceEventHandler;
 import org.folio.inventory.dataimport.handlers.actions.CreateItemEventHandler;
+import org.folio.inventory.dataimport.handlers.actions.CreateMarcAuthoritiesEventHandler;
```
Comment (psmagin) [inline src/main/java/org/folio/inventory/dataimport/consumers/DataImportKafkaHandler.java:21]:
```suggestion
import org.folio.inventory.dataimport.handlers.actions.CreateMarcAuthorityEventHandler;
```
Maybe will be better to use singular?

File: src/main/java/org/folio/inventory/dataimport/handlers/actions/CreateMarcAuthoritiesEventHandler.java [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+    return false;
+  }
+
+  private JsonObject prepareAuthority(DataImportEventPayload dataImportEventPayload) {
+    var authorityAsJson = new JsonObject(dataImportEventPayload.getContext().get(AUTHORITY.value()));
+    if (authorityAsJson.getJsonObject(AUTHORITY_PATH) != null) {
+      authorityAsJson = authorityAsJson.getJsonObject(AUTHORITY_PATH);
+    }
+    //Uncomment when
+//    authorityAsJson.put(SOURCE_KEY, MARC_FORMAT);
```
Comment (psmagin) [inline src/main/java/org/folio/inventory/dataimport/handlers/actions/CreateMarcAuthoritiesEventHandler.java:145]: When we should uncomment it? Do we have a story for this?

Comment (natali-zaitseva) [reply src/main/java/org/folio/inventory/dataimport/handlers/actions/CreateMarcAuthoritiesEventHandler.java:145]: yes MODINVSTOR-825, I will update the comment with ticket number

Comment (psmagin) [reply src/main/java/org/folio/inventory/dataimport/handlers/actions/CreateMarcAuthoritiesEventHandler.java]: But the ticket that you mentioned is for mod-inventory-storage. If we wait for the MODINVSTOR-825 competition, we also should have a mod-inventory ticket to uncomment this part of code.

Comment (natali-zaitseva) [reply src/main/java/org/folio/inventory/dataimport/handlers/actions/CreateMarcAuthoritiesEventHandler.java]: created an issue to not forgot about uncommenting this line https://issues.folio.org/browse/MODINV-576

**Review by psmagin — APPROVED** [review]

**Review by Igor-Gorchakov — APPROVED** [review]

**Review by oleksandr-dekin — APPROVED** [review]

**Review by RuslanLavrov — APPROVED** [review]

**Pull request merged**